modin/pandas/test/test_groupby.py

```diff
@@ -1968,6 +1968,21 @@ def compare(obj1, obj2):
     compare(reference_by, result_by)
 
 
+@pytest.mark.parametrize("width", [2**5, 2**10])
+@pytest.mark.parametrize("height", [2**5, 2**10])
+@pytest.mark.parametrize("concat_length", [5, 10])
+def test_concat_groupby(width, height, concat_length):
+    # from https://github.com/modin-project/modin/issues/4464
+    data = np.random.randint(0, 100, size=(width, height))
+    df = pd.DataFrame(data)
+    big_df = pd.concat([df for _ in range(concat_length)])
+    big_groupby = big_df.groupby(1)
+
+    big_groupby.count()
+    big_groupby.sum()
+    big_groupby.min()
```

Comment:
```suggestion
    big_groupby.count()
```

We should only test the function that caused the error.